bindings/python/setup.py

```diff
@@ -36,12 +37,19 @@ def deps_list(*pkgs):
 extras["numpy"] = deps_list("numpy")
 extras["tensorflow"] = deps_list("tensorflow")
 extras["jax"] = deps_list("jax", "flax")
+extras["paddlepaddle-gpu"] = deps_list("paddlepaddle-gpu")
```

Comment:
Why the `gpu` version? We don't require `tensorflow-gpu` so by pure symetric we shouldn't install the paddle-gpu either.

Cpu is fine for testing anyway (and if users have the gpu version, everything should work, correct ?)

Reply:
you are right, we donot need paddlepaddle-gpu